packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -133,7 +141,7 @@ export class GCFBootstrapper {
       return {...config, Octokit: LoggingOctokit} as Options;
     } else {
       logger.info('custom logging instance not enabled');
-      return config as Options;
+      return {...config, Octokit} as Options;
```

Comment:
always use the `Octokit` constructor, I believe this will  help us avoid diamond dependency issues.

Reply:
what are "diamond dependency issues"?

Reply:
we were bumping into an issue where probot wanted `@octokit/rest@16.x`, whereas our library `release-please` wanted `@octokit/rest@18.x`, I don't think we need to worry about two versions of this dependency in the tree any longer.

Reply:
Probot v10 only upgraded to v17 to ease the transition from v16, as you'd only need to address all deprecation messages in order to upgrade. Probot v11 will have `@octokit/rest` v18. If that's a problem I can cut a Probot v11 release sooner, no big deal

Reply:
@gr2m I would love to get on `v18` ideally, in testing so far I've been able to use `v18` as the dependency.